src/js/pages/Index.js

```diff
@@ -146,19 +145,20 @@ var Index = React.createClass({
 
   handleWindowResize(event) {
     const windowWidth = event.target.innerWidth;
-    const { isVisible } = this.internalStorage_get();
 
-    if (windowWidth <= MOBILE_THRESHOLD && isVisible) {
-      SidebarActions.close();
+    if (isViewportChangingTo(previousWidth, windowWidth) === viewport.DESKTOP) {
```

Comment:
I spent some time talking to @Fabs about how we could do this in a good way state isn't everywhere.

the idea was to build something to be re-used.